tools/mafft/mafft.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="rbc_mafft" name="MAFFT" version="7.221.1">
+<tool id="rbc_mafft" name="MAFFT" version="7.221.2">
     <description>Multiple alignment program for amino acid or nucleotide sequences</description>
     <requirements>
         <requirement type="package" version="7.221">mafft</requirement>
```

Comment:
Maybe a good time to require the latest 7.271 version? In this case, tool_dependencies.xml should be updated accordingly.